sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineEMSILMSIOperationsTests.java

```diff
@@ -254,7 +255,9 @@ public void canCreateUpdateVirtualMachineWithEMSI() {
     }
 
     @Test
+    @LiveOnly
     public void canCreateVirtualMachineWithLMSIAndEMSI() {
+        // LiveOnly because "test timing out after latest test proxy update"
```

Comment:
Is this time out for PLAYBACK only, or time out for both?

If PLAYBACK only, use `@DoNotRecord(skipInPlayback = true)` (`@LiveOnly` won't run in record mode)

Reply:
Fixed in the new version.

Reply:
Answer? Is this time out for PLAYBACK only, or time out for both?

Reply:
This is time out for PLAYBACK only.